tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -7,6 +7,8 @@
  * @bug         No known bugs
  */
 
+#include <chrono>
```

Comment:
I thought that we have generally used C header files although the file extension is `*.CPP`.  :)
Do you wanna use `chrono `of CPP to use clock and time duration?

Reply:
Yes, `chrono` is used measure time duration. 
I used `chrono` because it is the recommended way for C++
However to keep consistency, I will update to `time.h`

Reply:
IMO, g_get_real_time() also possible way.

Reply:
updated to use `g_get_real_time()`